plugins/woocommerce/client/admin/client/settings-payments/components/buttons/settings-button.tsx

```diff
@@ -18,25 +24,33 @@ interface SettingsButtonProps {
 	 */
 	buttonText?: string;
 	/**
-	 * ID of the plugin that is being installed.
+	 * Whether the associated provider plugin is being installed.
 	 */
-	installingPlugin: string | null;
+	isInstallingPlugin?: boolean;
```

Comment:
I took the liberty of renaming this field as it would provide the gateway ID only before the respective plugin's installation (null after that).